### PR TITLE
Rename context swapping to context extension. Fix some grammar issues…

### PR DESCRIPTION
**Pull request by Nick-Lucas**
… in the documentation

Closes #

## 🎯 Changes

What changes are made in this PR? Is it a feature or a bug fix?

## ✅ Checklist

- [ ] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

File: www/blog/2022-11-21-announcing-trpc-10.mdx [hunk 1]
```diff
@@ -41,7 +41,7 @@ With v10, you can:
 
 In v10, we've revisited the syntax for how you define your backend procedures, opening up more opportunities to bring in your desired logic in healthy ways. This version of tRPC features:
 
-- [Reusable middlewares](https://trpc.io/docs/v10/middlewares) with [Context Swapping](https://trpc.io/docs/v10/middlewares#context-swapping)
+- [Reusable middlewares](https://trpc.io/docs/v10/middlewares) with [Context Extension](https://trpc.io/docs/v10/middlewares#context-swapping)
```
Comment (sachinraja) [inline www/blog/2022-11-21-announcing-trpc-10.mdx:44]: should change the link here too

File: www/docs/server/middlewares.md [hunk 2]
```diff
@@ -111,11 +111,11 @@ export const appRouter = router({
 });
 ```
 
-## Context Swapping
+## Context Extension
 
-Context swapping in tRPC is a very powerful feature that allows you to create base procedures that dynamically infers new context in a flexible and typesafe manner.
+"Context Extension" in tRPC is a very powerful feature of middlewares, where you can dynamically add and override keys on a base procedure's context in a typesafe manner.
```
Comment (sachinraja) [inline www/docs/server/middlewares.md:116]:
```suggestion
"Context Extension" enables middlewares dynamically add and override keys on a base procedure's context in a typesafe manner.
```

File: www/blog/2022-11-21-announcing-trpc-10.mdx (hunk 1 above)
Comment (Nick-Lucas) [reply www/blog/2022-11-21-announcing-trpc-10.mdx]: Good spot

File: www/docs/server/middlewares.md [hunk 3]
```diff
@@ -111,11 +111,11 @@ export const appRouter = router({
...
 
-Below we have an example of a middleware that changes properties of the context, and procedures will receive the new context value:
+Below we have an example of a middleware that changes properties of a context, the change(s) are then available to all chained consumers, such as other middlewares and procedures:
```
Comment (sachinraja) [inline www/docs/server/middlewares.md:118]:
```suggestion
Below we have an example of a middleware that changes properties of a context, the changes are then available to all chained consumers, such as other middlewares and procedures:
```

"properties" is already plural and I think it's clear enough that multiple changes and a singular change will function in the same way

**Review by sachinraja — APPROVED** [review]

**Pull request merged**